tools/ci/retry.ps1

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env pwsh
+
+if ($args.Length -lt 1) {
+    Write-Host "Usage: $($MyInvocation.InvocationName) <command> [arguments]"
+    Write-Host "The script will execute the provided commands and retry in case of failures"
+    Exit 1
+}
+
+$NB_RETRIES = 5
+$command = [string]::Join(' ', $args)
+
+for ($i = 1; $i -le $NB_RETRIES; $i++) {
+    Invoke-Expression $command
+    $errorCode = $LASTEXITCODE
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ is this a builtin variable for PS?

Reply:
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_automatic_variables?view=powershell-7.4#lastexitcode yes